test/server/cards/15.4-TToTS/MushinNoShin.spec.js

```diff
@@ -0,0 +1,119 @@
+                this.initiateConflict({
+                    ring: 'fire',
+                    attackers: [this.monger],
+                    defenders: [this.doomedShugenja]
+                });
+                this.player2.pass();
+                this.player1.clickCard(this.monger);
+                this.player1.clickCard(this.doomedShugenja);
+                expect(this.player2).not.toBeAbleToSelect(this.mushin);
+            });
+
+            it('should prompt the player when targeted with their own abilities', function() {
+                this.noMoreActions();
+                this.initiateConflict({
+                    ring: 'fire',
+                    attackers: [this.monger],
+                    defenders: [this.nitenMaster]
+                });
+                this.player2.clickCard(this.castle);
+                this.player2.clickCard(this.nitenMaster);
+                expect(this.player2).not.toHavePrompt('Triggered Abilities');
+                expect(this.player2).not.toBeAbleToSelect(this.mushin);
+
+            });
+
+            it('should not prompt the player when targeted with their own abilities', function() {
+                this.player2.player.optionSettings.cancelOwnAbilities = true;
+                this.noMoreActions();
+                this.initiateConflict({
+                    ring: 'fire',
+                    attackers: [this.monger],
+                    defenders: [this.nitenMaster]
+                });
+                this.player2.clickCard(this.castle);
+                this.player2.clickCard(this.nitenMaster);
+                expect(this.player2).toHavePrompt('Triggered Abilities');
+                expect(this.player2).toBeAbleToSelect(this.mushin);
+
+            });
+
+            it('should prompt to cancel opponent\'s abilities', function() {
+
+                this.noMoreActions();
+                this.initiateConflict({
+                    ring: 'fire',
+                    attackers: [this.monger],
+                    defenders: [this.nitenMaster]
+                });
+                this.player2.pass();
+                this.player1.clickCard(this.monger);
+                this.player1.clickCard(this.nitenMaster);
+                expect(this.player2).toHavePrompt('Triggered Abilities');
+
+            });
+
+            it('chat', function() {
+
+                this.noMoreActions();
+                this.initiateConflict({
+                    ring: 'fire',
+                    attackers: [this.monger],
+                    defenders: [this.nitenMaster]
+                });
+                this.player2.pass();
+                this.player1.clickCard(this.monger);
+                this.player1.clickCard(this.nitenMaster);
+                expect(this.player2).toHavePrompt('Triggered Abilities');
+                this.player2.clickCard(this.mushin);
+                expect(this.getChatLogs(10)).toContain('player2 plays Mushin no Shin to cancel the effects of Young Rumormonger');
+
```

Comment:
Add a prompt check here (player2 should have "Conflict Action Window") to ensure that it actually cancels properly